packages/react-composites/scripts/runBrowserTests.mjs

```diff
@@ -282,9 +282,10 @@ function parseArgs(argv) {
   return args;
 }
 
-function notBetaBuild() {
+function needPreprocess() {
   const flavor = getBuildFlavor();
-  return flavor !== 'beta' && flavor !== 'beta-release';
+  console.log('current flavor:' + flavor);
+  return flavor !== 'stable';
```

Comment:
is it okay that this is not going to be true in the beta-release flavor? I thought that we needed to also make sure that we are running the pre-processing in that flavor as well?

Reply:
Because we change the function name to needPreprocess() so I reverse the logic a bit outside of this function call

Reply:
If the fn name is `needPreprocess` should this be
```suggestion
  return flavor !== 'beta';
```
As all others flavors that are not our default (beta) need preprocessed